Source/ACE.Database/WorldDatabase.cs

```diff
@@ -395,10 +404,12 @@ public PointsOfInterest GetCachedPointOfInterest(string name)
         }
 
         /// <summary>
-        /// This takes under 1 second to complete.
+        /// Retrieves all points of interest from the database and adds/updates the points of interest cache entries with every point of interest retrieved.
         /// </summary>
-        public void CacheAllPointsOfInterest()
+        /// <returns>time used</returns>
+        public TimeSpan CacheAllPointsOfInterest()
```

Comment:
I'd prefer this function not return the duration it took to do the work.

If a consumer of the function is interested in that information, it should calculate itself.